engines/router/missionctl/fiberapi/routing_strategy.go

```diff
@@ -3,7 +3,9 @@ package fiberapi
 import (
 	"context"
 	"encoding/json"
+	"strings"
 
+	"github.com/buger/jsonparser"
```

Comment:
Grabbed this from one of Turing Router's existing package dependencies: https://github.com/caraml-dev/turing/blob/main/engines/experiment/go.mod#L7